tidy3d/plugins/invdes/design.py

```diff
@@ -11,11 +12,10 @@
 from tidy3d.components.autograd import get_static
 
 from .base import InvdesBaseModel
+from .postprocess import CustomPostprocessOperation, PostProcessFnType, PostprocessOperationType
```

Comment:
We should decide for either `PostProcess` or `Postprocess`. My vote goes to `PostProcess`

Reply:
I agree I prefer `PostProcess`